src/gtk/slider.cpp

```diff
@@ -297,6 +296,7 @@ bool wxSlider::Create(wxWindow *parent,
     m_scrollEventType = GTK_SCROLL_NONE;
     m_needThumbRelease = false;
     m_blockScrollEvent = false;
+    m_tickFreq = 0;
```

Comment:
It looks like this is left uninitialized when default ctor is used, which is relatively harmless but still not ideal as `GetTickFreq()` will return garbage until `Create()` is called.

Reply:
Sorry for the slow reply. Hmm, the only way I can see around this is to add the initialisation to both the default ctor and `Create()` because adding a default value inside the class declaration won't work on older compilers.

Reply:
The standard way used in wx is to have a private `Init()` method called from all ctors, including the default one.

Reply:
Yay, thanks - good strategy!